client/src/graphics/animated_sprite.lua

```diff
@@ -100,13 +95,9 @@ function AnimatedSprite:update()
     end
   end
 end
-
-  function AnimatedSprite:draw(x, y, rot, x_scale, y_scale)
-    GraphicsUtil.drawBatch(self.animations[self.currentAnim].spriteSheet, self.animations[self.currentAnim].frames[self.currentFrame][1], x, y, rot, x_scale, y_scale)
-  end
```

Comment:
Given the nature of singular animations, it is very doubtful that a manual spritebatch approach helps.
Within a single animation it is generally not expected to draw more than one frame at a time, so manually batching will not help draw performance because there will normally only be a single frame in the batch and manually batching also breaks the autobatching love does on its own. I believe its wise to rely on that doing its job for us for now.
We could consider adding batch support by passing in a spriteBatch from the outside rather than an image to better support drawing the same animation many times on screen but at the moment it seems overkill and as mentioned, the autobatcher should already take care for that for the most part.
As the references to the spriteSheet were also unused and GraphicsUtil.drawBatch does not exist, I have removed them.